std_msgs/Header header
string robot_name
int8 last_inter_id
int8 curr_inter_id
RobotInfo robot_info
---
bool success
string message